README.md

```diff
@@ -1,3 +1,78 @@
+# Chitter!
+
+This is Makers weekend challenge no.4 where we design a basic twitter clone based on the user stories below. The Chitter app allows users to post 'peeps'.
+
+## User Stories
+
+The following user stories were implemented as part of this project:
+
+```
+As a Maker
+So that I can let people know what I am doing  
+I want to post a message (peep) to chitter
+```
+
+```
+As a maker
+So that I can see what others are saying  
+I want to see all peeps in reverse chronological order
+```
+
+```
+As a Maker
+So that I can better appreciate the context of a peep
+I want to see the time at which it was made
+```
+
+```
+As a Maker
+So that I can post messages on Chitter as me
+I want to sign up for Chitter
+```
+
+## How to use
+
+#### To set up the project
+
+Clone this repo and then run 
+
+```
+bundle install
+```
+
+#### To set up the database
+
+Connect to `psql` and create the `chitter` and `chitter_test` databases:
+
+```
+CREATE DATABASE chitter;
+CREATE DATABASE chitter_test;
+```
+To set up the appropriate tables, connect to each database in `psql` and run the SQL scripts in the `db/migrations` folder in the given order.
+
```

Comment:
I appreciate these clear instructions. 

Using these, I was able to get it set this up on my local machine to run Rspec, which showed all tests passing 🥇